README.md

```diff
@@ -14,6 +14,10 @@ For convenience, you can also access the most recent list of providers in the fo
 
 - [https://providers.optimade.org/providers.json](https://providers.optimade.org/providers.json)
 
+Additionally, there is a providers dashboard that includes the results of validation of listed implementations. This can be accessed at the following URL:
```

Comment:
One sentence, one line, is nice for future git diffs.
```suggestion
Additionally, there is a providers dashboard that includes the results of validation of listed implementations.
This can be accessed at the following URL:
```